themes/daux-blue/config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I'm not sure why github identified this as a new file but it's a copy of the original.